terminal/TerminalServer.cpp

```diff
@@ -534,14 +522,28 @@ void startJumpHostClient() {
 }
 
 int main(int argc, char **argv) {
-  SetVersionString(string(ET_VERSION));
-  ParseCommandLineFlags(&argc, &argv, true);
-  google::InitGoogleLogging(argv[0]);
-  GoogleLogFatalHandler::handle();
-  GOOGLE_PROTOBUF_VERIFY_VERSION;
-  FLAGS_logbufsecs = 0;
-  FLAGS_logbuflevel = google::GLOG_INFO;
-  srand(1);
+  // easylogging parse verbose arguments, see [Application Arguments]
+  // in https://github.com/muflihun/easyloggingpp/blob/master/README.md
+  START_EASYLOGGINGPP(argc, argv);
+  // GFLAGS parse command line arguments
+  ParseCommandLineFlags(&argc, &argv, false);
+
+  // Easylogging configurations
```

Comment:
Move this into LogHandler and pass the filename, verbosity, etc. as arguments to some function in LogHandler